rules/opentitan/fpga_cw310.bzl

```diff
@@ -37,6 +37,13 @@ load("//rules/opentitan:toolchain.bzl", "LOCALTOOLS_TOOLCHAIN")
 _TEST_SCRIPT = """#!/bin/bash
 set -e
 
+function cleanup {{
+  echo "cleanup: {post_test_harness} {post_test_cmd}"
+  {post_test_harness} {post_test_cmd}
+}}
+
+trap cleanup EXIT
```

Comment:
We might want to skip this if there's nothing to do for cleanup?

Reply:
We could skip entirely but it would make the code more complex for little gain?